src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -887,7 +887,7 @@ public enum VerticalLineAlignment
     /// <summary>
     /// An enum of ellipsis position.
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
     public enum EllipsisPosition
     {
         /// <summary>
```

Comment:
End, Start, and Middle enums should also be opened.

Reply:
@wonrst , done